scripts/test/SceneBuilder.ps1

```diff
@@ -0,0 +1,300 @@
+   $UnityPath = ${Env:${{ parameters.UnityVersion }}}
```

Comment:
Instead of using environmental values, we usually push these in as explicit arguments to the script - otherwise there are basically "hidden" inputs.